conscious/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "conscious",
+  "assets": [
+    {
+      "description": "Cvn is a Layer-1 blockchain built to deliver on the promise of DeFi",
+      "denom_units": [
+        {
+          "denom": "acvnt",
+          "exponent": 0
+        },
+        {
+          "denom": "cvnt",
+          "exponent": 18
+        }
+      ],
+      "base": "acvnt",
+      "name": "Cvn",
+      "display": "cvnt",
+      "symbol": "CVN",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/conscious/images/cvn.svg",
```

Comment:
Again, these images (which the png is square while the svg is not) seem unintentionally low quality/pixelated. Do we have clearer images?